content/en/agent/guide/cluster-agent-custom-metrics-server.md

```diff
@@ -0,0 +1,322 @@
+---
+title: Custom Metrics Server for the Cluster Agent
+kind: guide
+disable_toc: true
+further_reading:
+- link: "agent/kubernetes/cluster/"
+  tag: "Documentation"
+  text: "Datadog Cluster Agent"
+---
+
+## Introduction
+
+Kubernetes v1.2 introduced [Horizontal Pod Autoscaling][1]. This allows users to autoscale off of basic metrics like `CPU`, it but requires a resource called `metrics-server` to run alongside your application.
+As of Kubernetes v1.6, it is possible to autoscale off of [custom metrics][2].
+Custom metrics are user defined and are collected from within the cluster.
+Kubernetes v1.10, introduced support for external metrics, so that users can autoscale off of any metric from outside the cluster that is collected for you by Datadog.
+
+The custom and external metric providers, as opposed to the metrics server, are resources that have to be implemented and registered by the user.
+
+As of v1.0.0, the Custom Metrics Server in the Datadog Cluster Agent implements the External Metrics Provider interface for External Metrics.
+This guide explains how to set it up and how to autoscale your Kubernetes workload based off of your Datadog metrics.
+
+## Requirements
+
+1. Running Kubernetes >v1.10 in order to be able to register the External Metrics Provider resource against the API Server.
+2. Having the Aggregation layer enabled. Refer to the [Kubernetes aggregation layer configuration documentation][3] to learn how to enable it.
+
+## Walkthrough
+
+### Preliminary disclaimer
+
+Autoscaling over external metrics does not require the Node Agent to be running—you only need the metrics to be available in your Datadog account. Nevertheless, this guide describes autoscaling an NGINX deployment based off of NGINX metrics, collected by a Node Agent.
+
+The following assumptions are made:
+1. You have Node Agents running (ideally from a DaemonSet) with the Autodiscovery process enabled and functional.
+2. Agents are configured to securely (see [the security premise section][4] of the official documentation) communicate with the Datadog Cluster Agent.
```

Comment:
Instead of breaking the sentence with the `(..)` could we put the further-reading-link about security as an extra sentence of as a subordinate of the first one?